packages/client/modules/outcomeCard/components/OutcomeCardAssignMenu/TaskFooterTeamAssigneeMenu.tsx

```diff
@@ -33,9 +33,13 @@ const TaskFooterTeamAssigneeMenu = (props: Props) => {
   }
 
   return (
-    <Menu {...menuProps} ariaLabel={'Assign this task to another team'}>
+    <Menu
+      {...menuProps}
+      defaultActiveIdx={taskTeamIdx}
+      ariaLabel={'Assign this task to another team'}
+    >
       <DropdownMenuLabel>Move to:</DropdownMenuLabel>
-      {assignableTeams.map((team) => {
+      {teams.map((team) => {
```

Comment:
Instead of filtering `teams`, I thought we could include the team the task is currently assigned to. In production it currently shows "Move to" menu with no items in the menu if there are no other teams.

This PR shows this currently assigned team. But, unlike the `/team` page, clicking the currently assigned team doesn't unassign it; the update function doesn't fire.